### DISCUSSION TITLE
Sorulara cevap vericek Admin sıkıntısı varmı?

### DISCUSSION THREAD

**Discussion by Abdullahoda** [IBAX-io/go-ibax#1245] [Soru Cevap - Türkçe] [answered]
Sanırım bu bir sorun bizi Telegram ve githup ta yalnız bırakmış gibisiniz..

**Comment by Abdullahoda** [comment, marked as answer]
Herkes birbirine soru soruyor Admin eksikliği mevcut..

**Reply by Yusufknk** [reply]
Github yöneticileri sizlere cevap veriyor

**Comment by Trajedi032** [comment]
Yeterince bilgilendirme yok proje için daha çok bilgi almak istiyorum

**Reply by Yusufknk** [reply]
İbax anasayfasında tüm bilgiler mevcuttur. Lütfen detaylı inceleyiniz.

**Comment by Omertastan2036** [comment]
Admin sallamiyor
